syntax = "proto3";
import "google/protobuf/empty.proto";

message FileInfo {
    string name = 1;
    int64 size = 2;
    string timestamp = 3;
}

message FilesResponse {
    repeated FileInfo file_info = 1;
}

message FindFileRequest {
    string file_name = 1;
}

service FileServices{
    rpc ListFiles(google.protobuf.Empty) returns (FilesResponse) {}
    rpc FindFile(FindFileRequest) returns (FilesResponse);
}
